server/sonar-web/src/main/js/app/components/nav/global/global-nav-menu.js

```diff
@@ -77,9 +77,13 @@ export default React.createClass({
   },
 
   renderAdministrationLink () {
-    if (!window.SS.isUserAdmin) {
+    // FIXME check permissions
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)